### DISCUSSION TITLE
Use of LAMMPS input file starting simulation from restart file (instead of data file)

### DISCUSSION THREAD

**Discussion by felipe-pace-evaristo** [MICCoM/SSAGES-public#50] [Q&A]
I'm getting the following error trying to run SSAGES with the 22Aug2018 version of LAMMPS. The LAMMPS input file SSAGES calls starts its simulation from a restart file:

```
ERROR: Invalid flag in header section of restart file (../read_restart.cpp:930)
Last command: read_restart  soft_pushoff.restart.10000
```

Is SSAGES currently able to operate on LAMMPS simulations starting from restart files? If so, how can I make that work?

**Comment by p-zubieta** [comment]
Hi @felipepacci. Can you share your restart file? In principle one should be able to restart a LAMMPS simulation.

**Reply by felipe-pace-evaristo** [reply]
You can find the files for this run here: https://www.dropbox.com/scl/fo/c72ngre7x0ik52jg18n3d/h?rlkey=aukrwmycrwqu6o11tcrvaxban&dl=0

The restart file I am trying to use is `soft_pushoff.restart.10000`. The LAMMPS input file to be used during the ABF run is `PGNs_implicit_solvent_equilibration_nvt_abf.lmp`. `PGNs_implicit_solvent_equilibration_nvt_abf.lmp` currently inputs a data file via the `read_data` command, but I'd prefer to input the restart file `soft_pushoff.restart.10000` via the `read_restart` command instead.
